### PR TITLE
Use go:embed for static yaml files

### PR DESCRIPTION
**Pull request by jsafrane**
Move from bindata to go:embed.

Contains bump of library-go and related changes.

### CODE REVIEW

**Comment by jsafrane** [comment]
/hold
for https://github.com/openshift/library-go/pull/1117
/retest

**Comment by jsafrane** [comment]
/hold cancel
updated with openshift/library-go#1117

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -121,38 +127,47 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
...
-		generated.MustAsset("vsphere_cloud_config.yaml"),
+		cloudConfigBytes,
 		kubeClient,
 		kubeInformersForNamespaces,
 		operatorClient,
 		configInformers,
 		controllerConfig.EventRecorder,
 	)
 
+	scBytes, err :=assets.ReadFile("storageclass.yaml")
```
Comment (tsmetana) [inline pkg/operator/starter.go:163]: Nit: missing space.

Comment (jsafrane) [reply pkg/operator/starter.go:163]: This should have been caught by make verify!

Comment (jsafrane) [reply pkg/operator/starter.go]: It is caught by `make verify`, we just don't run it in CI :-)
Fixed.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**